### ISSUE TITLE
isTimed: false not functioning?

### ISSUE THREAD

**Issue by paqpaqpaq** [jacobgelman/homebridge-gpio-valve#1]
When I enter
"isTimed": false,
it will be disregarded. The timer will start no matter what.

What could be the reason that only the faucet and irrigation icons are showing?

**Comment by ladvoc** [comment]
It appears the Home app currently only supports these two icons; either iOS 12 or a supplemental update to iOS 11 may add support for the rest.

As for the `isTimed` option being disregarded, this appears to be a bug. I will perform some testing to determine its cause.

**Comment by ladvoc** [comment]
The bug was the result of a simple misuse of the logical or operator and has now been fixed. I will create a new release which addresses this bug as well as another I noticed within the next few days. However, if you would like to use the patched version now, please install the plugin directly from GitHub:
```
npm install git+https://github.com/jacobgelman/homebridge-gpio-valve.git -g
```

**Comment by paqpaqpaq** [comment]
Thanks!
No problem. I’ll wait for the update.

**Comment by ladvoc** [comment]
This bug is now fixed in [version 2.0](https://github.com/jacobgelman/homebridge-gpio-valve/releases/tag/v2.0). You can install it now through NPM.
